fcgl/forum
# Add Testing dependencies and implementation

## Problem
Don't have testing framework set up

## Solution
Implement testing framework and add one unit test

## Testing
1. Build and Run: `docker-compose up --build`
2. Confirm that the application built successfully. Application won't build if a test fails

Should see the following in the build logs:

<img width="601" alt="Screen Shot 2019-08-13 at 2 55 49 PM" src="https://user-images.githubusercontent.com/30189498/62943867-b5608580-bddb-11e9-99aa-061f76a6a2db.png">
